posydon/binary_evol/CE/step_CEE.py

```diff
@@ -214,23 +214,61 @@ def __call__(self, binary):
                                 'stripped_He_Core_He_burning']:
             # system merges
             binary.state = 'merged'
-            if binary.event in ["oCE1", "oDoubleCE1"]:
-                binary.event = "oMerging1"
-            if binary.event in ["oCE2", "oDoubleCE2"]:
-                binary.event = "oMerging2"
+            if double_CE:
+                binary.event = 'DCE_merger'
```

Comment:
maybe make it obvious it is at the onset/start of the process of merging (not just in general a merger product), by making it "DCE_merger_start", consistent with your implementation of "*_start" in this PR